src/main/app/claims/models/claim.ts

```diff
@@ -467,6 +469,10 @@ export class Claim {
       return false
     }
 
+    if (this.state === 'BUSINESS_QUEUE') {
```

Comment:
I would follow the pattern that we set and make this a method. See the isTransferred() method

Reply:
done